Dockerfile

```diff
@@ -41,6 +41,9 @@ COPY Makefile .
 RUN make update
 
 COPY . .
+
+RUN go mod tidy
```

Comment:
didn't you encounter some errors stuff if not used cmd `go mod tidy` ?

Reply:
Good catch! it's really will encounter some errors, maybe the reason is that we updated golang to v1.6.x